### PR TITLE
adds timezonechange problem.

### LINKED ISSUE #7: Change the system's timezone to the same of SFO, USA
problem name: `timezonechange`

### PR DESCRIPTION
**Pull request by Priyankasaggu11929**
Fixes #7.

### CODE REVIEW

File: lymproblems/timezonechange.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+def setup():
+    "Setup timezonechange"
+    pass  # Nothing to do.
+
+
+def verify():
+    "Verify timezonechange"
+
+    if os.popen("date +%T").read().strip("\n") != datetime.now(pytz.timezone("US/Pacific")).strftime("%H:%M:%S"):
```
Comment (kushaldas) [inline lymproblems/timezonechange.py]: Verify it via system level, which link needs to be changed to `US/Pacific` for this to be working?

Remember, here we are verifying the status of the system.

**Review by kushaldas — CHANGES_REQUESTED** [review]

File: lymproblems/timezonechange.py (hunk 1 above)
Comment (Priyankasaggu11929) [reply lymproblems/timezonechange.py]: @kushaldas I made the changes as asked. Please review it again.

File: lymproblems/timezonechange.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    pass  # Nothing to do.
+
+
+def verify():
+    "Verify timezonechange"
+
+    SFOtimezone = "/usr/share/zoneinfo/US/Pacific"
+    localtimezone = "/etc/localtime"
+
+    if os.popen("zdump {}".format(localtimezone)).read().strip(
```
Comment (kushaldas) [inline lymproblems/timezonechange.py]: Here is the function to use <https://docs.python.org/2/library/os.path.html#os.path.realpath>

**Review by kushaldas — CHANGES_REQUESTED** [review]

File: lymproblems/timezonechange.py (hunk 2 above)
Comment (Priyankasaggu11929) [reply lymproblems/timezonechange.py]: @kushaldas I added the above function.

**Review by kushaldas — APPROVED** [review]
Looks okay to me, thank you.

**Pull request merged**